rpc_util.go

```diff
@@ -345,6 +345,30 @@ func recv(p *parser, c Codec, s *transport.Stream, dc Decompressor, m interface{
 	return nil
 }
 
+type rpcStats struct {
```

Comment:
How about rpcInfo instead?  Stats makes me think there are metrics inside.  Combine "stats" with bytesSent/Received, and I expect those to be ints instead of bools.